### DISCUSSION TITLE
利用AI預測憂鬱症程度

### DISCUSSION THREAD

**Discussion by liangyinglly** [1101-GS4719-C/Python-Community-GS4719-C#653] [General]
剛剛看了一位同學分享利用AI預測憂鬱症。
因為身邊有人有為憂鬱症所苦，所以上網查了一下資料。
發現史丹佛大學「以人為本AI研究院（Stanford Human-Centered AI Institute, HAI）」有一群年輕學生提出一種能自動檢測憂鬱症的機器學習方法，來衡量憂鬱症嚴重程度，藉此提高診斷準確性與普及性。
他們提出多重模態方法（multi-modal method），使用現代手機常見的3D面部表情和口說語言等技術來判斷患者的憂鬱症程度，也就是說這個技術結合了語句辨識、電腦視覺和自然語言處理。
這些技術都是AI的應用，也是通識中心都有開的通識課，是很好的學習資源。
https://smctw.tw/3906/
就我觀察我身邊的憂鬱症患者，因為這種精神上的病症有太多變化與可能的表徵，每個人都症狀都不盡相同。
因此以電腦視覺、語音文字等技術來作心理病況的評估預測仍需要大量的範本與數據來讓機器學習。

**Comment by leon10456** [comment]
感謝樓主分享

**Comment by Lai27330** [comment]
對憂鬱症這種心理情緒的掌握，心理醫生的幫助還是遠大於AI的，而且像是情緒這種無法量化的狀態也沒辦法找到一定的標準來衡量輕重，所以應該還是要透過專家和AI的結合才有實質效果。

**Comment by HaoFish123** [comment]
感謝分享

**Comment by treeann** [comment]
AI真的太多用途了.....

**Comment by stan890425** [comment]
感謝大大分享

**Comment by jcpan821** [comment]
現在疫情都關在家，好像罹患憂鬱症的比例變高了

**Comment by alexthebigpp** [comment]
AI感覺用來治療心理疾病可以蠻準確的。

**Comment by Ben6uo** [comment]
那樣就可以拯救很多很多的人了

**Comment by yenwei12138** [comment]
利用面部表情和口說語言來分析學習的AI除了可以用在診斷憂鬱症，感覺在其他領域也很有搞頭(如教育方面

**Comment by bloodtrail0511** [comment]
感覺要拿來測一下室友，他怪怪的，發起瘋來就只用英文講話==

**Reply by yucheng1028** [reply]
太可怕了八 QAQ

**Comment by etin2468** [comment]
雖然可以預測，但治療還是要真正的心理醫師

**Comment by Lynn12378** [comment]
我覺得情緒是機器最難以去理解分析的部分
